src/lib.rs

```diff
@@ -0,0 +1,450 @@
+extern crate sxd_document;
+
+use std::str;
+use std::string::String;
```

Comment:
You don't have to `use` these, they are in the global namespsace.

Reply:
Ah, good to know. Thanks!